index.js

```diff
@@ -1,17 +1,53 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    let index = 0;
+    while (index < this.length && item > this.items[index]) {
+      index++;
+    }
+    this.items.splice(index, 0, item);
+    this.length++;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos < 0 || pos >= this.length) {
+      throw new Error("OutOfBounds");
+    }
+    return this.items[pos];
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return this.items[this.length - 1];
+  }
 
-  min() {}
+  min() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return this.items[0];
+  }
 
-  sum() {}
+  sum() {
+    let total = 0;
+    for (let i = 0; i < this.length; i++) {
+      total += this.items[i];
+    }
+    return total;
```

Comment:
Esto esta perfect!, pero si queremos usar un poco de programacion mas funcional yo te diria de hacerlo con el reduce, te pongo como seria y vas a ver lo facil que se lee (repito esta solucion que has puesto es perfecta, solo te estoy enseñando otras formas de hacerlo)
```suggestion
    const result = this.items.reduce((accumulator, currentValue) => accumulator + currentValue, 0);
    return result;
```
Bien, el propio metodo reduce retorna el valor por defecto que se le haya puesto cuando el array esta vacio, es decir que si el array esta vacio obtendriamos en la variable result un 0, que es justo lo que queremos :D